# deep_learning_for_single_cell_analysis

## [Surveys](#content) 
1. **Eleven grand challenges in single-cell data science** [2020, 318] <br/>
2. **Deep learning tackles single-cell analysis: A survey of deep learning for scRNA-seq analysis** [2021, 0] <br/>
3. **Emerging deep learning methods for single-cell RNA-seq data analysis (mini review)** [2019, 21] <br/>
4. **Graph representation learning for single-cell biology** [2021, 1] <br/>
5. **Graph Neural Networks and Their Current Applications in Bioinformatics** [2021, 0] <br/>


## [A. Spatial Single Cell Analysis](#content) 
**Intro videos** <br/>
a. [Next generation tools for spatial genomics](https://www.youtube.com/watch?v=HmYuS1qHuew) 2021.3.27 - Fei Chen, Ph.D., Broad Institute of MIT and Harvard <br/>
b. [Deep learning to integrate histology with spatial transcriptomics](https://www.youtube.com/watch?v=9PYeMn8t57Y) 2020.11.23 - James Zou, AP at Stanford <br/>
c. [Integrated spatial genomics reveals invariant features in the nucleus](https://spatial.caltech.edu/2021/10/15/presentation-video-integrated-spatial-genomics-reveals-invariant-features-in-the-nucleus/) 2021.10.15 - Long Cai, AP at Caltech <br/>

**Intro**: [Spatially Resolved Single-Cell Data in 2021: where are we?](https://scipio.bio/blogs/spatially-single-cell-data-2021/) <br/>
**Latest update** [Spatial Transcriptomics from 10xgeomics](https://www.10xgenomics.com/spatial-transcriptomics) <br/>
1. **DSTG: deconvoluting spatial transcriptomics data through graph-based artificial intelligence** [2021, 15, Briefings in Bioinformatics] <br/>
   [[paper]](https://academic.oup.com/bib/article/22/5/bbaa414/6105942)[[code]](https://github.com/Su-informatics-lab/DSTG)
2. **Method of the Year: spatially resolvedtranscriptomics** [2021, 76, Nature Methods] <br/>
   [[paper]](https://www.nature.com/articles/s41592-020-01033-y)
3. **Inferring spatial and signaling relationships between cells from single cell transcriptomic data** [2020, 42, Nature Communications] <br/>
   [[paper]](https://www.nature.com/articles/s41467-020-15968-5)[[code]](https://github.com/zcang/SpaOTsc)
4. **Annotation of Spatially Resolved Single-cell Data with STELLAR** [2021, 0, preprint] <br/>
   [[paper]](https://www.biorxiv.org/content/10.1101/2021.11.24.469947v1)
5. **Characterizing spatial gene expression heterogeneity in spatially resolved single-cell transcriptomics data with nonuniform cellular densities** [2021, 4, Genome Research] <br/>
   [[paper]](https://genome.cshlp.org/content/early/2021/05/25/gr.271288.120)[[code]](https://github.com/JEFworks-Lab/MERINGUE) <br/>
   a. [mOB Spatial Transcriptomics Analysis](https://github.com/JEFworks-Lab/MERINGUE/blob/master/docs/mOB_analysis.md) <br/>
   b. [Multi-section 3D Breast Cancer Spatial Transcriptomics Analysis](https://github.com/JEFworks-Lab/MERINGUE/blob/master/docs/BCL_analysis.md) <br/>
   c. [3D Drosophila Spatial Transcriptomics Analysis](https://github.com/JEFworks-Lab/MERINGUE/blob/master/docs/drosophila_3D_analysis.md) <br/>
   d. [Understanding MERINGUE's Spatial Cross-Correlation Statistic using Simulations](https://github.com/JEFworks-Lab/MERINGUE/blob/master/docs/simulation.md) <br/>
   e. [Spatially-informed transcriptional clustering with MERINGUE](https://github.com/JEFworks-Lab/MERINGUE/blob/master/docs/spatial_clustering.md) <br/>
6. **Integrating spatial gene expression and breast tumour morphology via deep learning** [2020, 61, Nature Biomedical Engineering] <br/>
   [[paper]](https://www.nature.com/articles/s41551-020-0578-x)[[code]](https://github.com/bryanhe/ST-Net)


## [B. Tasks in ST Data Analysis](#content) 
**B.1 Cell type identification** <br/>
   There are two general approaches for estimating cell type composition: <br/>
   a. to evaluate the enrichment of cell type-specific markers among the experessed genes at each spot <br/>
   b. deconvolution, to quantitatively eastimate the proportion of different cell types at each location <br/>

1. **Comprehensive mapping of tissue cell architecture via integrated single cell and spatial transcriptomics** [2020] <br/>
   [[paper]](https://www.biorxiv.org/content/10.1101/2020.11.15.378125v1.full.pdf)[[code]](https://github.com/BayraktarLab/cell2location)
2. **DEEPsc: a deep learning-based map connect- ing single-cell transcriptomics and spatial imaging data.** [2021] <br/>
3. **GLUER: integrative analysis of single-cell omics and imaging data by deep neural network.** [2021] <br/>




 

## [Application of Single Cell Analysis](#content) 

### [a. Cancer Research and Single Cell Analysis](#content) 
#### [Surveys](#content) 
1. **Advancing Cancer Research and Medicine with Single-Cell Genomics** [2020, 61] <br/>
2. **The Human Tumor Atlas Network: Charting Tumor Transitions across space and Time at single cell resolution** [2020, 138] <br/>
3. **Single-cell profiling of tumor heterogeneity** [2021, 19] <br/>
4. **Applications of single-cell sequencing in cancer research** [2021, 7] <br/>
5. **Single-cell analysis for cancer immunotherapy** [2021, 1] <br/>
6. **Single‚Äêcell RNA sequencing in cancer research** [2021, 12] <br/>
7. **Leveraging Single-Cell Approaches in Cancer Precision Medicine** [2021, 3] <br/>
8. **Single-cell analysis in cancer genomics** [2015, 151] <br/>






